ui/src/components/userProfile/UserProfileDesktop.js

```diff
@@ -128,6 +138,7 @@ const UserProfileDesktop = (props) => {
 
   useEffect(async () => {
```

Comment:
async function in useEffect is an anti-pattern. May want to remove the awaits or use a self-invoking function